orbitx/flight_gui.py

```diff
@@ -32,73 +34,37 @@
-        ctrl_c_handler: function = signal.getsignal(signal.SIGINT)
         # Note that this might actually start an HTTP server!
-        #import vpython
+        assert len(physical_state_to_draw.entities) >= 1
 
         ################################ Attributes ############################### 
 
-        self._vpython: module = vpython        
-
-        if self._vpython.__version__ == '7.4.7':
-            # vpython installs "os._exit(0)" signal handlers in 7.4.7.
-            # Very not good for us.
-            # Reset the signal handler to default behaviour.
-            signal.signal(signal.SIGINT, ctrl_c_handler)
-
-            # os._exit(0) is in one other place, let's take that out too.
-            # Unfortunately, now that vpython isn't calling os._exit, we have
-            # to notify someone when the GUI closed.
-            # Note to access vpython double underscore variables, we have to
-            # bypass normal name mangling by using getattr.
-            def callback(*_):
-                getattr(self._vpython.no_notebook, '__interact_loop').stop()
-                self.closed = True
-            self.closed = False
-            getattr(self._vpython.no_notebook,
-                    '__factory').protocol.onClose = callback
-
-        # Set up our vpython canvas and other internal variables
-        self._scene = vpython.canvas(
-            title='<b>OrbitX</b>',
-            align='right',
-            width=800,
-            height=600,
-            center=vpython.vector(0, 0, 0),
-            up=vpython.vector(0, 0, 1),
-            forward=vpython.vector(0.1, 0.1, -1),
-            autoscale=True
-        )
-
+        self._last_physical_state: protos.PhysicalState = physical_state_to_draw
+        # self._vpython: module = vpython
+        self._vpython = vp.vp()        
+        # self._vpython_version_check() # check whether its version is 7.4.7
+        vp.vpython_version_check() # check whether its version is 7.4.7
+        self._scene: vpython.vpython.canvas  = self._init_canvas() # create a vpython canvas object
         self._show_label: bool = True
         self._show_trails: bool = False
         self._pause: bool = False
-        self._scene.autoscale: bool = False
         self.pause_label: vpython.vpython.label = None
-        self._texture_path: NoneType = texture_path
+        self._texture_path: str = texture_path
         self._commands: list = []
         self._spheres: dict = {}    
         self._labels: dict = {}
         self._landing_graphic: dict = {}
-
-        self.pause_label = vpython.label(
-            text="Simulation Paused.", visible=False)
-
-        self._scene.bind('keydown', self._handle_keydown)
-        self._scene.bind('click', self._handle_click)
-        # Show all planets in solar system
-        self._scene.range: float = 696000000.0 * 15000  # Sun radius * 15000
+        self.pause_label: vpython.vpython.label = vpython.label(text="Simulation Paused.", visible=False)
```

Comment:
is this line longer than 80 characters? Have you run a pep8 formatter on this file?